presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -504,6 +504,11 @@ default void createView(ConnectorSession session, SchemaTableName viewName, Stri
         throw new PrestoException(NOT_SUPPORTED, "This connector does not support creating views");
     }
 
+    default void createView(ConnectorSession session, SchemaTableName viewName, String viewData, ConnectorTableMetadata viewMetadata, boolean replace)
```

Comment:
Why do we even need 2 methods? It feels like we should simply migrate.

Reply:
I think we should have a deprecation period since this is an spi change in case there are external connectors that need to migrate.

Reply:
We are trying to do our best to do not make any breaking changes, however I'm not sure if the delay is going to be really helpful. But from the other side it will make the SPI more messy. Anyhow, I don't have a strong opinion here. We can keep the old SPI method.

Reply:
That's fair.  People can just as easily wait to upgrade until they fix their connectors.  @ucalegon206, let's remove the deprecated method entirely